examples/fishjam-chat/app.json

```diff
@@ -45,7 +47,15 @@
           }
         }
       ],
-      ["./plugin/build/with-local-paths-for-native-packages.js"] // Only relevant for local development of packages/ios-client and packages/android-client
+      ["./plugin/build/with-local-paths-for-native-packages.js"],
+      [
+        "expo-build-properties",
```

Comment:
**suggestion:** Can you move `expo-build-properties` before our plugins? Just to make sure we override everything as the last step.